### PR TITLE
Fix aria attributes in ContextualMenu and CommandBar

### LINKED ISSUE #3952: Aria-owns property for ContextualMenu item for a sub menu is wrong
#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Already done, see #3951 

Requested priority: Normal

Products/sites affected: VSTS

#### Describe the issue:

If submenu props are given in a ContextualMenu item, one may give any ID for the generated submenu. However, the aria-owns property for the parent menu item will always be a generated ID rather than the one specified. This fix causes the aria-owns property to use the correct submenu ID.

#### Actual behavior:

Wrong aria-owns value

#### Expected behavior:

Not wrong aria-owns value

### PR DESCRIPTION
**Pull request by trevorsg**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #3952 and #3953
- [x] Include a change request file using `$ npm run change`

#### Description of changes

2 Bug fixes:

1. When a ContextualMenu creates a submenu, the aria-owns property on the parent menu item will be wrong if a submenu props object contained an ID.
2. The attributes aria-setsize and aria-posinset are missing from CommandBar. Microsoft Narrator requires these to announce progress through menu items.

#### Focus areas to test

Unit test included for (1) above. Otherwise this is a low-risk change.

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx [hunk 1]
```diff
@@ -110,14 +110,17 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
...
 
     return (
       <div className={ css('ms-CommandBar', styles.root, className) } ref='commandBarRegion'>
         { searchBox }
         <FocusZone ref='focusZone' className={ styles.container } direction={ FocusZoneDirection.horizontal } role='menubar' >
           <div className={ css('ms-CommandBar-primaryCommands', styles.primaryCommands) } ref='commandSurface'>
-            { renderedItems!.map((item, index) => (
-              this._renderItemInCommandBar(item, index, expandedMenuItemKey!)
+            { renderedItems!.map(item => (
+              this._renderItemInCommandBar(item, posInSet++, setSize, expandedMenuItemKey!)
```
Comment (manishgarg1) [inline packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx]: >posInSet [](start = 49, length = 8)

Can you please add another value to the snapshot tests such that the count increment is working correctly. Or a unit test would be equally good.

**Comment by manishgarg1** [comment]
Please merge master to make the screener tests pass

**Comment by trevorsg** [comment]
@manishgarg1 I am not sure the screener test failures are caused by my changes. I see the same failures in other runs.

File: packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx (hunk 1 above)
Comment (trevorsg) [reply packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx]: Not sure what you meant by "add another value to the snapshot tests such that the count increment is working correctly" so I added a unit test.

Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx]: That works! But he just meant add another item in the `items` of the command bar in the snapshot unit test which is easier to at least validate in the snapshot that the setSize works. :)

**Comment by dzearing** [comment]
Spinning off screener tests again. If it fails unexpectedly, please remerge master.

I think there are 2 issues; 

master was broken - this is now fixed.

screener has a bug which will link "officedev/master" instead of "trevorsg/master" which is hiding what broke. I talked with screener devs, and they're fixing this!

File: packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx (hunk 1 above)
Comment (trevorsg) [reply packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx]: Ok, thanks David!

**Pull request merged**